### DISCUSSION TITLE
Bug in searchbox results preview view

### DISCUSSION THREAD

**Discussion by ericsampson** [community/community#8592] [Other Feature Feedback, Questions, & Ideas] [answered]
There appears to be a missing space(s) between the end of the filename (e.g. `ApiController.cs`) and the repo name:

![image](https://user-images.githubusercontent.com/1183853/145418283-d2dab107-6d5f-4ea9-816f-789621d5bd2b.png)

**Comment by tclem** [comment, marked as answer]
Good find! I've pushed a fix for symbol suggestions that should work a bit better on long paths and symbol names, let me know how it looks.

**Reply by ericsampson** [reply]
That looks great, thanks Timothy! I'm excited to be able to use the new Code Search :)

**Reply by ericsampson** [reply]
Feel free to close/archive this discussion as appropriate now. Cheers
